fetch_coursier

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import sys
+
+if sys.version_info[0] < 3 or sys.version_info[1] < 5:
+    print("This program requires Python version 3.5 or newer")
+    sys.exit(1)
+
+import os
+import argparse
+from scalaplugin import install_coursier
+from common import is_existing_dir
+from logger import add_verbosity_arg
```

Comment:
These kinds of imports of local files have always made me a bit nervous, but now that there are a lot of Python files here, I'd really prefer that we do this right by making this a real Python package. At the moment, you have absolute imports to local files, which is problematic for the following reasons:

- You lose relocatability, in that these imports only work if this script is a sibling to the modules you are importing, or if you mess with the `PYTHONPATH`/`sys.path`.
- Namespacing-wise, you've used some very common English words as module names (`common`, `logger`), which occupy the top-level, global package namespace. This could easily clash with other libraries.

Can we package this as a proper Python package, complete with `setup.py`? See https://python-packaging.readthedocs.io/en/latest/minimal.html and https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html if you want some pointers to it.

More specifically, I think it'd be a good idea to create a more structured directory structure, such as:

```
api-scala-sifive/
    setup.py
    scalaplugin/
        __init__.py
        other_python_modules.py
    bin/
        fetch_coursier
```

Reply:
Done, I moved things to a directory called `lib`, if that's still to generic I can rename it.